doc/releases_and_migration/migration_guide_v280.rst

```diff
@@ -116,20 +116,20 @@ See the following example of device characteristics:
 New Sidewalk ID option
 ======================
 
-A new option has been introduced that allows fetching of the Sidewalk ID for the device. 
+A new option has been introduced that allows fetching of the Sidewalk ID for the device.
 This option is functional only after the device has been registered, as the Sidewalk ID is assigned post-registration.
 
 The Sidewalk ID can be retrieved using the ``-gsi`` subcommand of the CLI (DUT) option:
 
 .. code-block:: console
 
-  uart:~$ sid last_status 
+  uart:~$ sid last_status
   [00:00:51.950,347] <inf> sid_cli: Device Is registered, Time Sync Fail, Link status: {BLE: Down, FSK: Down, LoRa: Down}
   uart:~$ sid option -gsi
   [00:00:55.582,641] <inf> sid_cli: sid_option returned 0 (SID_ERROR_NONE); SIDEWALK_ID: BFFFFFABCD
 
 Fixing uplink buffer
 ====================
 
-Upon migration, you can remove the ``pending_message_list`` workaround, as the first uplink message after Link Up payload has been fixed in the v2.8.0 release. 
-For the issue details, see Amazon Sidewalk Known Issues.
+Upon migration, you can remove the ``pending_message_list`` workaround, as the first uplink message after Link Up payload has been fixed in the v2.8.0 release.
+For the issue details, see Amazon Sidewalk section of `Known issues for the nRF Connect SDK for the v2.8.0`_.
```

Comment:
@ktaborowski has this item been added to known issues?

Reply:
It should be rather removed from known issues, as it is already fixed. I think that adding a note in migration guide is a good idea, as the temporary workaround has been done in the application layer, and users should know it is no longer necessary

Reply:
I dont see this error in known issues list for amazon sidewalk 2.7.0. It seems we forgot to add it in documentation. So, no reference in this case.